src/tree/treemath.rs

```diff
@@ -136,7 +136,7 @@ pub(crate) fn dirpath_long(
 ) -> Result<Vec<NodeIndex>, TreeMathError> {
     let r = root(size);
     if index == r {
-        return Ok(vec![]);
```

Comment:
Please add a test for this.

Reply:
This can only be tested with interop test vectors, recalculating the same thing is a moot point. We should do that during the IETF hackathon.

Reply:
A simple test with predefined tree is certainly possible. That's all that's needed for this for now.
Of course we need more and better tests. But this code change should have a test. Same below.

Reply:
There's a test now in `treemath_test.rs`